base/applications/taskmgr/perfdata.c

```diff
@@ -247,6 +259,12 @@ void PerfDataRefresh(void)
     {
         BufferSize += 0x10000;
         pBuffer = (LPBYTE)HeapAlloc(GetProcessHeap(), 0, BufferSize);
+        if (!pBuffer)
```

Comment:
Failing in the case of this or pPerfData is reasonable.

Reply:
Made this case skippable and some others too